src/dmd/backend/cgobj.d

```diff
@@ -469,14 +469,14 @@ int insidx(char *p,uint index)
             mov     ECX,p - [ESP]           ;
         }
     else
-        asm
+        asm nothrow
```

Comment:
Why are these required for this PR? It feels like they should logically go in a separate one.

Reply:
Yeah they probably should.  This gets rid of some warnings, looks like the windows makefile ignored the warnings but the build.d isn't.  Will probably move these into another PR before this PR is ready.